docs/ABAC.mdx

```diff
@@ -9,7 +9,7 @@ ABAC is ``Attribute-Based Access Control``, meaning you can use the attributes (
 
 Use the official ABAC example for example:
 
-```ini
+```ini title="model.ini"
```

Comment:
Why add title? We need to systematically decide why, when and where to add title or not

Reply:
We might tackle this in another PR